src/frontend/src/views/Account/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { Button, Icon, InputText, Link } from "czifui";
+import { useRouter } from "next/router";
+import React, { ChangeEvent, useEffect, useState } from "react";
+import { H1, H2, P } from "src/common/styles/basicStyle";
+import AbandonChangesModal from "./components/AbandonChangesModal";
+import {
+  GrayIconWrapper,
+  StyledDivider,
+  StyledH3,
+  StyledHeaderRow,
+  StyledRow,
+  StyledSection,
+  SubText,
+  WhiteIconWrapper,
+} from "./style";
+
+export default function Account(): JSX.Element {
+  const router = useRouter();
+  // TODO:194969 - get user's current gisaid id for the default value
+  const [gisaidId, setGisaidId] = useState("");
+  const [hasUnsavedChanges, setUnsavedChanges] = useState<boolean>(false);
```

Comment:
```suggestion
  const [hasUnsavedChanges, setHasUnsavedChanges] = useState<boolean>(false);
```

we like to use a convention where the set is essentially identically named as get.

Reply:
ah! I do too, just saw some that weren't so I was trying to match.  Happy to change it.

Reply:
Yea, there are some legacy ones that dont match and we never updated them yet. feel free to change them whenever you see them!